code/model/editableformfields/EditableCheckboxGroupField.php

```diff
@@ -22,7 +22,7 @@ public function getFormField()
         // Set the default checked items
         $defaultCheckedItems = $this->getDefaultOptions();
         if ($defaultCheckedItems->count()) {
-            $field->setDefaultItems($defaultCheckedItems->map('EscapedTitle')->keys());
+            $field->setDefaultItems($defaultCheckedItems->map('EscapedValue')->keys());
```

Comment:
We probably shouldn't escape this value here as it should be escaped by the template instead - that logic is already in place (cut down example):

```
<% loop $Options %>
    <option value="$Value.XML"...
<% end_loop %>
```

Perhaps you could change this and the other implementations to just use `Value`? Would also be worth adding a test including some special characters in the values to ensure that this is working correctly, regardless of whether it's escaped here or in the template.